community/planning/rest_api/backwards_compatibility.md

```diff
@@ -0,0 +1,235 @@
+<!--
+  Copyright 2022 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+# REST API Backwards Compatibility Plan
+
+## Overview
+
+Endpoints can have a wide range of compatibility support. For example,
+consider the following list of endpoints:
+
+| Protocol Version | 1 | 2 | 3 | 4 | 5 |
+|------------------|---|---|---|---|---|
+| /endpoint1       | 1 | 1 | 1 | 1 | 5 |
+| /endpoint2       | 1 | 2 | 3 | 3 | 3 |
+| /endpoint3       | 1 | 1 | 1 | 4 | 5 |
+| /endpoint4       | - | - | 3 | 4 | 5 |
+
```

Comment:
It'd probably be good to add a short explanation of what the numbers for each endpoint mean.

Reply:
Updated! Added a more robust explanation of what is happening with these endpoints below.